n_i2c.c

```diff
@@ -297,3 +207,161 @@ bool i2cNoteReset()
     // Done
     return notecardReady;
 }
+
+/**************************************************************************/
+/*!
+  @brief  Receive bytes over I2C to the Notecard.
+  @param   buffer
+            A buffer to receive bytes into.
+  @param   size (in/out)
+            - (in) The size of the buffer in bytes.
+            - (out) The length of the received data in bytes.
+  @param   delay
+            Respect delay standard transmission delays.
+  @param   timeoutMs
+            The maximum amount of time, in milliseconds, to wait for serial data
+            to arrive. Passing zero (0) disables the timeout.
+  @param   overflow (out)
+            The contents did not fit inside the provided buffer.

```

Comment:
Same comment as before. It's more than just a bool, at least from what I can tell.